tests/profiling/exporter/test_pprof.py

```diff
@@ -329,6 +329,8 @@
             thread_native_id=123987,
             thread_name="MainThread",
             trace_id=1322219321,
+            trace_resource="myresource",
+            trace_type="http",
```

Comment:
```suggestion
            trace_type=SpanTypes.HTTP.value,
```
I know using `.value` makes little sense here, but this will get fixed by #2526

Reply:
I could probably switch to using `SpanTypes` too in `ddtrace.profiling.exporter.pprof`. I was reluctant to import more tracer code for maybe no good reason. WDYT?

Reply:
I think using `SpanTypes` makes it more explicit the meaning of `trace_type` by tying it to `Span` semantics. These values are unlikely to change, but if they do we'd have to make the change in just one source.

Reply:
> I was reluctant to import more tracer code for maybe no good reason. WDYT?

That module is just for constants, and basically all span types are. using the enum would be good if we ever need to fix something in the future, and we can ensure consistency between spans and profiles.


.... not a strong opinion, not sure I care enough, at least for tests

Reply:
yup I changed it